types/animation.d.ts

```diff
@@ -29,3 +29,5 @@ export class CompositeAnimation implements IAnimation {
 }
 
 export const WalkAnimation: AnimationFn;
```

Comment:
Missing `naturalSpeed` here.

Reply:
These three animation functions all need this property.

Reply:
How to add type definition of this property at this situation? I am not familiar with TypeScript.

Reply:
That is `AnimationFn & { naturalSpeed?: number }`.
And `WalkAnimation` should be `WalkingAnimtaion`.

Reply:
Fine👌